packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeClipboard/MT5TradeClipboard.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { WalletClipboard, WalletText } from '../../../../../components/Base';
+import EditIcon from '../../../../../public/images/ic-edit.svg';
+import './MT5TradeClipboard.scss';
+
+type TMT5TradeClipboardProps = {
```

Comment:
Cant we change the name, its confusing clipboard have `clipboard | password` variant